easybuild/easyblocks/b/binutils.py

```diff
@@ -57,6 +58,8 @@ def extra_options(extra_vars=None):
     def configure_step(self):
         """Custom configuration procedure for binutils: statically link to zlib, configure options."""
 
+        sysroot = build_option('sysroot')
```

Comment:
@boegel We made a mistake here, the default is  `None` but we use this later in `os.path.join()` so that can't be.

But should it be
```
if not sysroot:
  sysroot = os.path.sep
```
or
```
if not sysroot:
  sysroot = os.path.abspath('.').split(os.path.sep)[0]+ os.path.sep
```
(ref. https://stackoverflow.com/questions/48333999/how-to-get-the-filesystems-root-directory-in-python)

Reply:
fixed in #2159